identity_service/src/main/kotlin/org/kiva/identityservice/api/ApiController.kt

```diff
@@ -61,6 +62,20 @@ class ApiController constructor(
         }
     }
 
+    @PostMapping("/store", produces = [MediaType.APPLICATION_JSON_VALUE])
+    fun store(@Valid @RequestBody storeRequest: StoreRequest): Mono<ResponseEntity<out Any>> {
+        try {
+            val backend: IBackend = backendManager.getbyName("template")
```

Comment:
I'm wondering if we should allow passing in the backend via the url, I think that's the pattern we use other places

Reply:
For sure, that would match the existing pattern, but it's meaningless to have any other backend. It doesn't make sense to directly "store" things in other backends and, even if it did, we'd need a different route in order to handle a differently shaped request body, or else lose the type safety and automated runtime validation by using some generic base type like `any`. TLDR: it seems to me like passing in the backend via the url opens the door for errors without much (any?) functional benefit.

Reply:
Ah yes that makes sense, I was thinking in terms of old patterns